codewof/programming/content/en/driver-speed/question.md

```diff
@@ -0,0 +1,10 @@
+# Driver speed
+
+Write a function `check_speed(speed)` that checks if a driver is driving above the speed limit.
+
+- If the speed is less than or equal to 50, the function should **print** `Ok.`
+- If the speed is above 50, the function should **print** `Too fast!`
+- If the speed is 75 or higher then the function should *also* **print** `License suspended.`
```

Comment:
The UK English spelling of the noun `licence` has been un-fixed

Reply:
*facepalm* I checked what it was on the NZ transport website and then somehow managed to write this version instead... Will change